crates/pathfinder/src/rpc/v02.rs

```diff
@@ -320,7 +321,7 @@ mod tests {
         .await
         .unwrap();
 
-        let client = client(addr);
+        let client = TestClient::v01(addr);
```

Comment:
This test is for v0.2, the test will not fail but instead will test __v0.1__ callability because in both versions of the api the method names are the same.

Reply:
Hmm, but how was this testing the `v0.2` endpoint previously? As far as I can see `client()` just constructed a client for the root URL?

Reply:
It wasn't which is why I kept this.